components/user_settings/display/user_settings_display.tsx

```diff
@@ -81,7 +136,7 @@ export default class UserSettingsDisplay extends React.PureComponent {
         }
 
         this.prevSections = {
-            theme: 'dummySectionName', // dummy value that should never match any section name
```

Comment:
We want to leave the comment here, not above.
The comment describes the use of `dummySectionName` as a value for this variable.

Reply:
resolved